README.md

```diff
@@ -35,6 +35,21 @@ main(int argc, char **argv) {
 }
```

Comment:
Why is this now `README.md`? I like `readme.md`!

Reply:
reverted (I used `README.md` as per GitHub standards,  but,  don't care tbh)